pyproject.toml

```
@@ -2,13 +2,31 @@
 requires = ['setuptools', 'wheel', 'requests']
 build-backend = 'setuptools.build_meta'
 
-[tool.black]
+[tool.ruff]
+target-version = "py38"
 line-length = 125
 
-[tool.isort]
-no_sections = true
-force_alphabetical_sort = true
-combine_as_imports = true
-profile = "black"
-skip_gitignore = true
-line_length = 125
+[tool.ruff.lint]
+select = [
```

```suggestion
extend-select = [
```

not (non-blocking): Prefer `extend-select` to not overwrite linters considered "sane defaults". Same applies to `extend-ignore`.